app/views/accounts/_identity_verification.html.erb

```diff
@@ -19,6 +19,17 @@
   </div>
 </div>
 
+<% if @presenter.active_profile? && !@presenter.connected_to_initiating_idv_sp? %>
+  <%= render AlertComponent.new(type: :warning, text_tag: 'div', class: 'margin-bottom-2') do %>
+    <p class='margin-bottom-0'><%= t('account.index.verification.connect_idv_account.intro') %></p>
+    <% if @presenter.connect_to_initiating_idv_sp_url.present? %>
+      <p><%= link_to(t('account.index.verification.connect_idv_account.cta'), @presenter.connect_to_initiating_idv_sp_url) %></p>
+    <% else %>
+      <p><%= t('account.index.verification.connect_idv_account.cta') %></p>
+    <% end %>
```

Comment:
Nit: Using a line break rather than paragraphs might be a bit simpler here

```suggestion
  <%= render AlertComponent.new(type: :warning, class: 'margin-bottom-2') do %>
    <%= t('account.index.verification.connect_idv_account.intro') %><br />
    <% if @presenter.connect_to_initiating_idv_sp_url.present? %>
      <%= link_to(t('account.index.verification.connect_idv_account.cta'), @presenter.connect_to_initiating_idv_sp_url) %>
    <% else %>
      <%= t('account.index.verification.connect_idv_account.cta') %>
    <% end %>
```